src/lib/CMakeLists.txt

```diff
@@ -564,6 +568,8 @@ set(
     cqf
     uninitialized_vector
     ${Boost_CONTAINER_LIBRARY}
+    ${Boost_SYSTEM_LIBRARY}
```

Comment:
I started getting Linker errors in the `hyriseBenchmarkPlayground` (https://hyrise-ci.epic-hpi.de/blue/organizations/jenkins/Hyrise%2Fhyrise/detail/PR-1231/5/pipeline) without having modified it.

I am at loss how hyrise/Playground/MicroBenchmarks etc. ever linked successfully without being linked to boost::system/thread. Both libraries are referenced in libHyrise in the server part.

Similar for CMAKE_DL_LIBS which is explicitly linked to the Console and the Tests, but not to, e.g., the playground. Any ideas?

@mrks @Bensk1

Reply:
Mmh, is that a consistent problem? Two days ago, we still had some hickups with the CI server (disk space running full etc).

Reply:
I could produce that locally as well, but only in the gcc-7 build and only on this branch. clang was fine. Still don't know what happened there.

Reply:
If you can reproduce this somehow in a way where we can look into it, please open an issue.